From f4f9513f808fae515acf8253269a4eec9a667cd5 Mon Sep 17 00:00:00 2001
From: Ian McInerney <ian.s.mcinerney@ieee.org>
Date: Tue, 27 Aug 2024 11:49:28 +0100
Subject: [PATCH] Fix compilation with Boost 1.86

Boost 1.86 removed the boost::random dependency from boost::uuid, so
we need to include those headers on our own now to use the random
mersenne twister implementation.

(cherry picked from commit a9e115925a5168034f60d0fe1e7b369861f84b82)
---
 common/kiid.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/kiid.cpp b/common/kiid.cpp
index 4c7e8eb18c..80aea3e55d 100644
--- a/common/kiid.cpp
+++ b/common/kiid.cpp
@@ -25,6 +25,7 @@
 
 #include <kiid.h>
 
+#include <boost/random/mersenne_twister.hpp>
 #include <boost/uuid/uuid_generators.hpp>
 #include <boost/uuid/uuid_io.hpp>
 #include <boost/functional/hash.hpp>
-- 
2.45.2.windows.1

